package.json

```diff
@@ -58,24 +57,23 @@
     "babel-plugin-istanbul": "^5.1.4",
     "babel-plugin-transform-export-extensions": "^6.22.0",
     "babel-preset-minify": "^0.5.0",
-    "chai-as-promised": "^7.1.1",
     "chai": "^4.2.0",
+    "chai-as-promised": "^7.1.1",
     "cross-env": "^5.2.0",
+    "eslint": "^4.19.1",
     "eslint-config-airbnb": "^16.1.0",
     "eslint-import-resolver-webpack": "^0.11.1",
     "eslint-plugin-import": "^2.17.3",
     "eslint-plugin-jest": "^22.21.0",
     "eslint-plugin-jsx-a11y": "^6.2.3",
-    "eslint-plugin-react": "^7.17.0",
-    "eslint": "^4.19.1",
+    "eslint-plugin-react": "^7.29.2",
     "mocha": "^6.1.4",
-    "npm-audit-ci": "^1.2.1",
     "nyc": "^14.1.1",
-    "sinon-chai": "^3.3.0",
     "sinon": "^7.3.2",
+    "sinon-chai": "^3.3.0",
     "vows": "^0.8.3",
+    "webpack": "^4.39.3",
     "webpack-cli": "^3.3.10",
-    "webpack-node-externals": "^1.7.2",
-    "webpack": "^4.39.3"
+    "webpack-node-externals": "^1.7.2"
```

Comment:
This is a big step forward; I think next on the list is clearing some of the current audit warnings.

I know it's a non-goal of this PR to bump dependencies, and `npm audit --production` only points to `yargs`

```
# additional detail from:
$ npm outdated --long | sort -k6
Package                                Current   Wanted  Latest  Location       Package Type     Homepage
regenerator-runtime                     0.13.2   0.13.2  0.13.9  @easypost/api  dependencies
source-map-support                      0.5.12   0.5.12  0.5.21  @easypost/api  dependencies     https://github.com/evanw/node-source-map-support#readme
superagent                               5.0.6    5.0.6   7.1.1  @easypost/api  dependencies     https://github.com/visionmedia/superagent
core-js                                  3.1.3    3.1.3  3.21.1  @easypost/api  dependencies     https://github.com/zloirock/core-js#readme
yargs                                   13.2.4   13.2.4  17.3.1  @easypost/api  dependencies     https://yargs.js.org/
@easypost/eslint-config-easypost-base    1.0.3    1.0.3   2.2.2  @easypost/api  devDependencies
@pollyjs/adapter-node-http               5.1.1    5.1.1   6.0.4  @easypost/api  devDependencies
@pollyjs/core                            5.1.1    5.1.1   6.0.4  @easypost/api  devDependencies
@pollyjs/persister-fs                    5.1.1    5.1.1   6.0.4  @easypost/api  devDependencies
eslint                                  4.19.1   4.19.1  8.10.0  @easypost/api  devDependencies  https://eslint.org
eslint-config-airbnb                    16.1.0   16.1.0  19.0.4  @easypost/api  devDependencies  https://github.com/airbnb/javascript
babel-loader                             8.0.6    8.0.6   8.2.3  @easypost/api  devDependencies  https://github.com/babel/babel-loader
eslint-import-resolver-webpack          0.11.1   0.11.1  0.13.2  @easypost/api  devDependencies  https://github.com/benmosher/eslint-plugin-import/tree/master/resolvers/webpack
babel-plugin-istanbul                    5.2.0    5.2.0   6.1.1  @easypost/api  devDependencies  https://github.com/istanbuljs/babel-plugin-istanbul#readme
nyc                                     14.1.1   14.1.1  15.1.0  @easypost/api  devDependencies  https://github.com/istanbuljs/nyc#readme
eslint-plugin-jest                     22.21.0  22.21.0  26.1.1  @easypost/api  devDependencies  https://github.com/jest-community/eslint-plugin-jest#readme
cross-env                                5.2.1    5.2.1   7.0.3  @easypost/api  devDependencies  https://github.com/kentcdodds/cross-env#readme
webpack-node-externals                   1.7.2    1.7.2   3.0.0  @easypost/api  devDependencies  https://github.com/liady/webpack-node-externals
webpack                                 4.46.0   4.46.0  5.69.1  @easypost/api  devDependencies  https://github.com/webpack/webpack
webpack-cli                             3.3.12   3.3.12   4.9.2  @easypost/api  devDependencies  https://github.com/webpack/webpack-cli#readme
mocha                                    6.2.3    6.2.3   9.2.1  @easypost/api  devDependencies  https://mochajs.org/
sinon                                    7.5.0    7.5.0  13.0.1  @easypost/api  devDependencies  https://sinonjs.org/
```

Reply:
I absolutely agree. The task of modernizing this lib was so vast that I'm chunking it down into as small of pieces as makes sense. Once this build system is out of the way, it opens the doors for fixing all the deps which is my next plan. Once that's done it should be buildable on 16+